app/src/main/java/uk/gov/android/authentication/login/AppAuthSession.kt

```diff
@@ -21,12 +23,29 @@ class AppAuthSession(
 
     override fun finalise(
         intent: Intent,
+        appIntegrity: Pair<String?, String?>,
```

Comment:
I think either this needs a specific data class to hold the Attestation and PoP or it needs to be well documented on the `LoginSession` interface please. At the moment it's not super clear what this is and what the `first` and `second` in the pair are supposed to be

Reply:
Sorry, I meant to change that and forgot, done